src/components/gltf-model-plus.js

```diff
@@ -202,6 +203,7 @@ function cachedLoadGLTF(src, preferredTechnique, onProgress) {
 AFRAME.registerComponent("gltf-model-plus", {
   schema: {
     src: { type: "string" },
+    basePath: { type: "string", default: undefined },
```

Comment:
If basePath is not specified, nothing checks for that case and ultimately will wind up with an error being thrown in `THREE.GLTFLoader` in `cachedLoadGLTF()`. There should probably be a check to catch that case prior to undefined being passed into `THREE.GLTFLoader`.